server/services/interventionsService.ts

```diff
@@ -50,6 +50,12 @@ export interface ServiceProvider {
   name: string
 }
 
+export interface SentReferral {
+  id: string
+  // At some point we’ll want to add in all the fields of a DraftReferral here
```

Comment:
can we use https://www.typescriptlang.org/docs/handbook/advanced-types.html#intersection-types for this purpose?

Reply:
Yeah, I didn't address it here because also need to figure out how to make everything non-nullable, possibly one of the [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html) would help but I haven't looked at them in detail. I can take a look into it tomorrow but didn't want to sink much time into it yet.

Reply:
I've done this now.